### PR TITLE
Admin Updates

### PR DESCRIPTION
**Pull request by alexander2001i**

### CODE REVIEW

File: packages/admin/src/graphql/schema.graphql [hunk 1]
```diff
@@ -4,6 +4,38 @@ schema {
   subscription: Subscription
 }
 scalar Date
+enum CartNetwork {
```
Comment (watchtower314) [inline packages/admin/src/graphql/schema.graphql]: is this `Cart` on purpose, or a typo? 😅

Comment (alexander2001i) [reply packages/admin/src/graphql/schema.graphql]: typo, thanks :D

File: packages/firebase/functions/src/core/graph/schema/cards/enums/CardNetwork.enum.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { enumType } from 'nexus';
+
+export const CardNetworkEnum = enumType({
+  name: 'CartNetwork',
```
Comment (watchtower314) [inline packages/firebase/functions/src/core/graph/schema/cards/enums/CardNetwork.enum.ts]: Same here

File: packages/firebase/functions/src/core/graph/schema/payments/queries/GetPayment.query.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+export const GetPaymentQuery = extendType({
+  type: 'Query',
+  definition(t) {
+    t.field('payment', {
+      type: PaymentType,
+      args: {
+        id: nonNull(idArg())
+      },
+      resolve: (root, args) => {
+        // @todo !important Do some kind of permission check !!!!!!!!!
```
Comment (watchtower314) [inline packages/firebase/functions/src/core/graph/schema/payments/queries/GetPayment.query.ts:14]: 😅

File: packages/firebase/functions/src/core/graph/schema/statistics/statistics.ts [hunk 4]
```diff
@@ -21,14 +24,61 @@ export const StatisticsType = objectType({
 
     t.int('newDiscussionMessages', {
       description: 'The amount of new discussion messages, send on that date'
-    })
+    });
+
+    t.int('commons', {
+      resolve: async () => {
+        // @todo This is bad, bad, bad and expensive
```
Comment (watchtower314) [inline packages/firebase/functions/src/core/graph/schema/statistics/statistics.ts:31]: 😅

**Review by watchtower314 — APPROVED** [review]
Looking good! I left some comments though, and I trust you'll fix eslint :)

File: packages/firebase/functions/src/core/graph/schema/payments/queries/GetPayment.query.ts (hunk 3 above)
Comment (alexander2001i) [reply packages/firebase/functions/src/core/graph/schema/payments/queries/GetPayment.query.ts:14]: This won't happen soon, but is needed :D

**Pull request merged**